piyushgarg-dev/piyushgargdev-nextjs
# fix: text color of 'Connect with me' heading

## What does this PR do?

Changes the color to green of heading 'connect with me' in contact section as like all the main headings of page. (others are yarn updates).

Fixes #1105 

![Connect with me](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/121534215/382211c9-56bf-47a7-863f-f5a27e180004)


## Type of change

Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to the home page
Scroll down to the end of the page
See the heading 'Connect with me'.

## Mandatory Tasks

✅ Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
